docs/releases/1.11.txt

```diff
@@ -325,6 +325,12 @@ URLs
 
 * ...
 
+Utilities
```

Comment:
In past releases, we haven't included changes in django.utils in the release notes.